Actions/AL-Go-Helper.ps1

```diff
@@ -310,9 +312,10 @@ function ReadSettings {
         "installPerformanceToolkit"              = $false
         "enableCodeCop"                          = $false
         "enableUICop"                            = $false
+        "doNotBuildTests"                        = $false
         "doNotRunTests"                          = $false
         "appSourceCopMandatoryAffixes"           = @()
-        "memoryLimit"                            = "6G"
+        "memoryLimit"                            = ""

```

Comment:
minor: IMO in almost all the cases memory usage is lower than 6GB but does that mean that we dont have any limits by default? what happens if a partner has a base app size app on github ?